server/raft_transport.go

```diff
@@ -159,11 +159,13 @@ func (t *rpcTransport) processQueue(nodeID roachpb.NodeID, storeID roachpb.Store
 	done := make(chan *gorpc.Call, cap(ch))
 	var req *storage.RaftMessageRequest
 	protoResp := &storage.RaftMessageResponse{}
+	raftIdleTimer := time.NewTimer(raftIdleTimeout)
 	for {
+		raftIdleTimer.Reset(raftIdleTimeout)
```

Comment:
`Reset` returns `false` if the timer had expired (in other words, it returns true if there's a value parked on the channel). Seems easy enough to mitigate:
```go
if raftIdleTimer.Reset(raftIdleTimeout) {
  <-raftIdleTimer.C
}
```